### ISSUE TITLE
5.95.2: connman/OE Addon forgets settings

### ISSUE THREAD

**Issue by hanzoh** [OpenELEC/OpenELEC.tv#4222]
Hello,

my HTPC with an i3-3225 worked fine via Wi-Fi on 5.0.8.
On a fresh 5.95.2 installation however, the wizard doesn't show any networks, so I have to skip that.
In the addon, I enabled wireless and disabled wired, chose a network and it connected (there is another bug I guess: when I choose the first entry and connect I have to enter the password. If the list refreshes in the background and another network is on top, it tries to use the password for that and thus is fails).

After a restart however, it disables wireless again and doesn't automatically connect. This prevents Kodi from using a MySQL database, even though I also activated "Wait for network", which isn't respected:

KODI: http://sprunge.us/YBAD
System: http://sprunge.us/jgZK
Journal: http://sprunge.us/YEFg

Kind regards,
hanzoh

**Comment by fritsch** [comment]
User did not include the information he was pointed to (documentation.md) when opening this bugreport.

**Comment by hanzoh** [comment]
Sorry, logs are there now.

**Comment by stefansaraev** [comment]
use wired.

and where is your network log ? I want to also see Filesystem.log

**Comment by hanzoh** [comment]
I really would use wired, if I could, but in this flat, it's just not possible.

Here are the network and filesystem logs:
network: http://sprunge.us/eZei
filesystem: http://sprunge.us/SCS

**Comment by stefansaraev** [comment]
everything looks good. no bug. please do:
```
ls -laR /storage/.cache/connman/ | pastebinit
cat /storage/.cache/connman/*/settings | pastebinit
```

**Comment by hanzoh** [comment]
I think I found the problem.
Ethernet was disabled in the BIOS while booting OE the first time.
Now I installed it again with Ethernet enabled, and it shows the connections in the wizard.
Seems like connman needs the wired connection to at least be there to initialize the wireless connections.

Wired is disabled now, and it automatically connects to wireless on reboot, but it still does not connect to the MySQL database during startup.
When setting the "Wait for network" to 30 or higher, then it still disabled "Wireless Networks".
Using a bash script in autostart.sh has the same effect, so even waiting 60 seconds does not result in a connection before Kodi trying to find the database.
